packages/components/src/components/hds/form/super-select/multiple/base.ts

```diff
@@ -109,7 +141,7 @@ export default class HdsSuperSelectMultipleBaseComponent extends PowerSelectComp
    * @default 'none'
    */
   get dropdownMaxWidthStyle() {
```

Comment:
Nit: missing this return type

Reply:
Good catch! Added in https://github.com/hashicorp/design-system/pull/2356/commits/08c99afae38f74beb19294c618baf00e1eb2ee5e